docker-entrypoint.d/39-oss-services.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/sh
+
+set -e
+
+NGINX_ROOT=/usr/share/nginx/html
+INITIALIZER_SCRIPT=$NGINX_ROOT/swagger-initializer.js
```

Comment:
@char0n I believe this should be `INITIALIZER_SCRIPT=$NGINX_ROOT/swagger-initializer-override.js`

Reply:
Everything should be now fixed. Did rounds of deep testing.

The mechanism works on the simple principle of piping new content to `swagger-initializer.js`. The moment we try to do anything else, copy or override the file we end up with original content from swagger-ui layer.